ui/app/AppLayouts/Chat/views/CommunitySettingsView.qml

```diff
@@ -296,6 +298,7 @@ StatusSectionLayout {
                 accounts: root.rootStore.accounts
 
                 onPreviousPageNameChanged: root.backButtonName = previousPageName
+                onSignMintTransactionOpened: communityTokensStore.computeDeployFee()
```

Comment:
This flow is just a base for backend integration and maybe it changes a bit, however it seems that to get this value it can take some time so it needs to be done in an asyncronous way to not blocking the UI. That's why here we are doing the request and we wait for a response in a connection.